packages/orbit-components/src/InputField/index.tsx

```diff
@@ -269,7 +269,7 @@ const InputField = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
           ref={ref}
           tabIndex={tabIndex !== undefined ? Number(tabIndex) : undefined}
           list={list}
-          aria-labelledby={!label ? inputId : undefined}
+          aria-label={label ? (label as string) : undefined}
```

Comment:
This is not adding `ariaLabel` prop. The component needs to accept the prop and add it, just like you did on Select.

Reply:
fixed. 🙈